lib/tip_tap/nodes/text.rb

```diff
@@ -22,7 +22,7 @@ def self.from_json(json)
       end
 
       def to_h
-        data = {type: type_name, text: text.presence || ""}
+        data = {type: type_name, text: text}
```

Comment:
IIRC I did the `presence || ""` bit because `text` was `nil` sometimes and TipTap JS wouldn't render the document stating that text can't be `null`. It turns out that text can't be an empty string either so my fix didn't actually work. I'm actually surprised that this is removing your `" "` string since that isn't `empty?`.

Reply:
Ah but it is `blank?` I guess so that is why it's falling back to this. I think your fix should be fine.

Reply:
We could also do `text || ""` to split the difference, but yeah `presence` is beging too aggressive here

Reply:
Yea that's a nice middle ground I think.